src/explore-education-statistics-frontend/src/modules/find-statistics/components/PublicationSummary.tsx

```diff
@@ -0,0 +1,70 @@
+import FormattedDate from '@common/components/FormattedDate';
+import VisuallyHidden from '@common/components/VisuallyHidden';
+import { PublicationSummaryWithRelease } from '@common/services/publicationService';
+import { releaseTypes } from '@common/services/types/releaseType';
+import styles from '@frontend/modules/find-statistics/components/PublicationSummary.module.scss';
+import Link from '@frontend/components/Link';
+import React from 'react';
+
+interface Props {
+  publication: PublicationSummaryWithRelease;
+}
+
+const PublicationSummary = ({ publication }: Props) => {
+  const {
+    legacyPublicationUrl,
+    latestRelease,
+    slug,
+    summary,
+    theme,
+    title,
+  } = publication;
+  return (
+    <li className={`${styles.container} govuk-!-margin-top-4`}>
+      {legacyPublicationUrl ? (
+        <>
+          <h3 className="govuk-!-margin-bottom-2">{title}</h3>
+          <p>
+            Currently available via{' '}
+            <a href={legacyPublicationUrl}>
+              Statistics at DfE <VisuallyHidden>for {title}</VisuallyHidden>
+            </a>
+          </p>
+        </>
+      ) : (
+        <>
+          <h3 className="govuk-!-margin-bottom-2">
+            <Link to={`/find-statistics/${slug}`}>{title}</Link>
+          </h3>
+          <p>{summary}</p>
+        </>
```

Comment:
In the case of legacy publications, wouldn't it be better to keep the link within the heading and strip out the link from 'Statistics at DfE'?

It currently seems quite inconsistent: ![image](https://user-images.githubusercontent.com/9917868/187676370-3219f31a-d6f1-462b-bd86-7ff5be65b29c.png)

Think it'd be a little confusing to users as to why the link has moved out of the heading. Users also don't really care if a link remains in a site or they get bounced somewhere else.

Moving it back into the title would have the side benefit of not needing extra visually hidden text.